submissions/antonkottans/hooli-style-popup/style.css

```diff
@@ -0,0 +1,172 @@
+* {
+	margin: 0px;
+	padding: 0px;
+}
+
+body {
+	min-width: 600px;
+	display: flex;
+	justify-content: flex-end;
+}
+
+ul {
+	list-style: none;
+}
+
+a,
+a:visited {
+	text-decoration: none;
+	color: #606060;
+}
+
+.popup-hr {
+	width: 100%;
+	margin: 20px 0;
+}
+
+.hidden-checkbox {
+	-webkit-appearance: none;
+	appearance: none;
+}
+
+.popup-checkbox-label {
+	cursor: pointer;
+	height: 32px;
+}
+
+.nav-menu {
+	display: flex;
+}
+
+.nav-menu-item {
+	padding: 20px;
+	width: 32px;
+	height: 32px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.nav-menu-item-link:focus,
+.nav-image:focus,
+.popup-checkbox:focus + .popup-checkbox-label {
+	outline: solid 2px #d0d0d0;
+}
+
+.nav-menu-item-link:hover,
+.nav-image:hover,
+.popup-checkbox-label:hover {
+	filter: drop-shadow(0px 0px 10px #e0e0e0);
+}
+
+.nav-image {
+	width: 32px;
+	height: 32px;
+}
+
+.popup-checkbox:checked ~ .popup-menu {
+	visibility: visible;
+	opacity: 1;
+}
+
+.popup-checkbox:checked ~ .popup-checkbox-label {
+	outline: 2px solid #d0d0d0;
+}
+
+.popup-menu {
+	border: solid 1px #d0d0d0;
+	position: absolute;
+	top: 80px;
+	right: 65px;
+	background-color: white;
+	box-shadow: 0px 0px 15px #aaaaaa;
+	visibility: hidden;
+	opacity: 0;
+	transition: opacity 200ms linear;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	overflow-y: overlay;
+	width: 362px;
+	height: auto;
+	max-height: 600px;
+	padding: 16px 0 0 0;
+}
+
+.popup-item {
+	padding: 5px;
+	width: 100px;
+	height: 100px;
+	display: flex;
+	justify-content: center;
+	align-items: flex-end;
+	flex-wrap: wrap;
+}
+
+.popup-item-img {
+	max-width: 48px;
+	max-height: 48px;
+	padding: 10px;
+	margin: auto 100%;
+	align-self: center;
+}
+
+a.popup-item:focus {
```

Comment:
Avoid styling elements using html tag as selector. [Reason#1](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) [Reason#2](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)
For base styles, it's ok but in main files/components, this is a bad practice.